web_external/util/annotationGeometryParser.js

```diff
@@ -155,7 +155,7 @@ class AnnotationGeometryContainer {
         // Look up ID of possibly existing geometry
         let geomId = this._getState(frame, trackId);
 
-        if (geomId) {
+        if (geomId !== undefined) {
             // Geometry already exists for the specified state; look it up and
             // modify it in place
             let geomToChange = this._frameMap(frame).get(trackId);
```

Comment:
I think this _framemap access is wrong

Reply:
In what way?

Reply:
Isn't _framemap is a map, so it needs **.get()** instead of **()** to get value?

Reply:
Ah, yes. (I'm sure there is a technical term for the brain's tendency to fail to notice a word that is expected to be present, but isn't, but I can't find it... anyway, fixed; thanks!)